specification/sql/resource-manager/Microsoft.Sql/preview/2021-02-01-preview/LedgerDigestUpload.json

```diff
@@ -0,0 +1,331 @@
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully disabled uploading ledger digests",
+            "schema": {
+              "$ref": "#/definitions/LedgerDigestUpload"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 LedgerInvalidDigestStorageTargetUpdateRequest - The database ledger digest storage upload request does not exist or has no properties object.\n\n * 400 InvalidDigestStorageEndpoint - The specified digestStorageEndpoint is invalid. It must be an Azure blob storage endpoint.\n\n * 400 MalformedDigestStorageEndpoint - The specified digest storage endpoint is a malformed URI.\n\n * 400 InvalidLedgerDigestUploadName - The ledger digest upload name is not supported. Please use the key name “current”.\n\n * 400 NoServerIdentity - No identity is configured for specified server. For information on how to configure an identity for the server see https://go.microsoft.com/fwlink/?linkid=2156804\n\n * 400 InsufficientStorageAccountPermissions - Read or write operations are not allowed on the storage account {0}.\n\n * 400 StorageAccountIsDisabled - The storage account {0} is disabled.\n\n * 400 AdalGenericError - The operation could not be completed because an Azure Active Directory error was encountered. The error message from Active Directory Authentication library (ADAL) is {0}.\n\n * 400 SubscriptionDoesNotHaveServer - Subscription {0} does not contain server {1}.\n\n * 400 InvalidStorageAccountName - The specified storage account is not valid or does not exist.\n\n * 404 SourceDatabaseNotFound - Specified database does not exist.\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found"
+          },
+          "202": {
+            "description": "Disabling the ledger digest upload is in progress."
+          }
+        },
+        "x-ms-examples": {
+          "Disables uploading ledger digests for a database": {
+            "$ref": "./examples/LedgerDigestUploadDisable.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LedgerDigestUploadProperties": {
+      "description": "The properties of a database ledger digest upload settings.",
+      "type": "object",
+      "properties": {
+        "digestStorageEndpoint": {
+          "description": "The digest storage endpoint, which must be either an Azure blob storage endpoint or an URI for Azure Confidential Ledger.",
+          "type": "string"
+        },
+        "state": {
+          "description": "Specifies the state of ledger digest upload.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "readOnly": true,
```

Comment:
Since state is read-only, it won't be able to be enforced by Azure policy. If that scenario matters, the state should be updatable via PATCH.

https://armwiki.azurewebsites.net/api_contracts/guidelines/policy.html#plcy003-limit-the-use-of-read-only-property-that-is-only-returned-after-initial-creation